modules/internal/ChapelDistribution.chpl

```diff
@@ -430,9 +430,9 @@ module ChapelDistribution {
     param strides: strideKind;
 
     // deprecated by Vass in 1.31 to implement #17131
-    //RSDW: @deprecated("domain.stridable is deprecated; use domain.strides instead")
+    @deprecated("domain.stridable is deprecated; use domain.strides instead")
```

Comment:
Kind nitpicky, but this would be much more readable without the newline between `@deprecated` and the proc

Reply:
Looking at our module code, there is always a newline between these two. So I will keep mine. Also my terminal windows are only 80 characters wide. :)

Reply:
😞 I realized my window was too small and made the spacing look really messed up. Nevermind